command/acl/token/update/token_update_test.go

```diff
@@ -72,21 +65,38 @@ func TestTokenUpdateCommand(t *testing.T) {
 
 	// update with node identity
 	t.Run("node-identity", func(t *testing.T) {
-		token := run(t, []string{
+		// create a token
+		token, _, err := client.ACL().TokenCreate(
```

Comment:
i wonder if i could move this token creation to a helper that just returns the token 🤔  maybe i am overcomplicating this

Reply:
It's certainly a possibility given how many times this construct appears in the tests.. but I think it's completely up to you to decide if you think it is worth factoring this out of the tests. It might be nice to be able to replace these occurrences with something like:

```go
token := createToken(t, aclToken, writeOpts)
```
And inside `createToken`, you could use `t` to make the necessary assertions. You still have to pass the token and write options from each test so it probably won't save a bunch of space.